src/streamlink/plugins/cinergroup.py

```diff
@@ -22,14 +23,42 @@
         |
         haberturk\.(?:com|tv)(?:/tv)?/canliyayin
         |
-        showmax\.com\.tr/canliyayin
+        bloomberght.com/tv
         |
-        showturk\.com\.tr/canli-yayin/showturk
+        showmax\.com\.tr/(canli-yayin|canliyayin)
+        |
+        showturk\.com\.tr/(canli-yayin|canliyayin)(/showtv)?
```

Comment:
The regex should be simplified and unnecessary capture groups be avoided.

```suggestion
        showmax\.com\.tr/canli-?yayin
        |
        showturk\.com\.tr/canli-?yayin(?:/showtv)?
```